src/Databases/DatabaseReplicatedWorker.cpp

```diff
@@ -196,6 +217,28 @@ void DatabaseReplicatedDDLWorker::initializeReplication()
             throw Exception(ErrorCodes::LOGICAL_ERROR, "Inconsistent database metadata after reconnection to ZooKeeper");
     }
 
+    bool first_initialization = active_node_holder == nullptr;
```

Comment:
We do `active_node_holder.reset();` in line 157, so will `first_initialization` always be true?